.gitignore

```diff
@@ -58,9 +57,6 @@ typings/
 # Yarn Integrity file
 .yarn-integrity
 
-# dotenv environment variables file
```

Comment:
Please restore those

Reply:
OkI can just include the one from the devcontainer